src/clamp.js

```diff
@@ -0,0 +1,27 @@
+var _curry3 = require('./internal/_curry3');
+
+/**
+ * Clamps a number within a range.
```

Comment:
Yes, I agree.

    Restricts a number to be within a range.

    Also works for other ordered types such as Strings and Dates.  (see Type::Ord)

That last bit it just a bit of fantasy, but something I'd love to be able to support.